### PR TITLE
Try 3 times

### PR DESCRIPTION
**Pull request by mbavaro**
@silverjam how would I go about testing this locally without rebuilding the entire image?

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018052114-7-gccd9b09
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=535462)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018052114-7-gccd9b09&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018052114-7-gccd9b09&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018052114-7-gccd9b09
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018052114-7-gccd9b09)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018052114-7-gccd9b09/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018052114-7-gccd9b09

**Comment by silverjam** [comment]
@mbavaro since this is in the overlay, a simple `make image` should pick up the change

File: board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga [hunk 1]
```diff
@@ -5,13 +5,22 @@ name="piksi_fpga"
 start() {
   bit="/lib/firmware/piksi_fpga.bit"
   if [ -f "$bit" ]; then
-    cat "$bit" > /dev/xdevcfg
-    prog_done=`cat /sys/devices/soc0/amba/f8007000.devcfg/prog_done`
-    if [ "$prog_done" != "1" ]; then
-      echo "ERROR: configuration failed"
-    fi
+    tries=1
+    while [ $tries -le 3 ]; do
```
Comment (silverjam) [inline board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga]: Need to increment tries somewhere

Comment (mbavaro) [reply board/piksiv3/rootfs-overlay/etc/init.d/S84piksi_fpga]: omg.. am I tired.. I started with a `for`, ended with a `while`, forgot the increment :(

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018052114-8-gcfc4453
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=535569)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018052114-8-gcfc4453&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018052114-8-gcfc4453&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018052114-8-gcfc4453
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018052114-8-gcfc4453)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018052114-8-gcfc4453/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018052114-8-gcfc4453

**Pull request merged**